src/perform_step/rosenbrock_perform_step.jl

```diff
@@ -612,13 +612,13 @@ end
 
 #### Rodas4 type method
 
-function initialize!(integrator, cache::Rodas4ConstantCache)
-  integrator.kshortsize = 2
-  integrator.k = typeof(integrator.k)(undef, integrator.kshortsize)
-  # Avoid undefined entries if k is an array of arrays
-  integrator.k[1] = zero(integrator.u)
-  integrator.k[2] = zero(integrator.u)
-end
+# function initialize!(integrator, cache::Rodas4ConstantCache)
```

Comment:
?

Reply:
This function is equivalent to the already defined `
function initialize!(integrator, cache::Union{...})` at line 261, except that `integrator.fsallast` and `integrator.fsalfirst` are not set here and remain `undef`. So firstly this dispatch is redundant and secondly it does not initialize compleately